CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+##[0.13.0] - 2022-10-14
+
+### Added
+ - Renovate configuration and replaced dependabot per OSPO recommendations
+
```

Comment:
* Since this change only affects the repository and not the published library. I generally do not include those changes in the CHANGELOG.
* The project has a [release process](https://wayfair-incubator.github.io/columbo/0.12.0/development-guide/#publishing-a-new-version) that automatically bumps the version number and updates the CHANGELOG for that release. Only the change description should be included in a feature PR.

Reply:
ok, this was not clear anywhere. Might be worth adding this to your contributing doc, as I added this due to the instructions there